help/md/fnFreeK.md

```diff
@@ -1,9 +1,52 @@
 ## name
 fnFreeK
 ## title
+Free K Rate Matrix
 ## description
+This function generates and returns a free rates matrix. 
 ## details
+This function accepts both a vector or a matrix of non-negative, real numbers in the 
+first argument to automatically generate a rate matrix with corresponding substitution 
+rates, returning a rate matrix object. The function will fill rates in the matrix from
+left to right as provided in the first argument, skipping the diagonal when using a vector
+as input. For this reason,using a vector of lengths 2 to 5 will create a 2-by-2 rate
+matrix but a vector of length 6 will create a 3-by-3 rate matrix as fnFreeK will have 
+enough values to fill the matrix. Using a matrix to create in fnFreeK will create a 
+rate matrix object with rates filled in their respective position in the provided matrix.
+Users can specify if matrix should be normalized in the second argument using a boolean 
+variable (default TRUE). Lastly users can specify what matrix exponential method to
+use (default eigen) with a string. Possible options include:
```

Comment:
Just two minor typos to fix in this paragraph: (1) in the phrase "Using a matrix to create in fnFreeK", there are either some missing words or the "in" doesn't belong there; and (2) there should be a space after the comma following "For this reason".

Reply:
Apologies, maybe I wasn't totally clear about what changes I was requesting. It was specifically this phrase, which I don't think is grammatical:

> Using a matrix to create in fnFreeK will create a rate matrix object with [...]

We could instead say something like:

- "Using a matrix in fnFreeK will create a rate matrix object with [...]"
- "Supplying a matrix to fnFreeK will create a rate matrix object with [...]"

Also, this is super minor, but if possible, let's remove the extra whitespace here (line 35):

```
               ↓
assigned using  their respective
```